### ISSUE TITLE
A11y_Visual Studio Code_HomePage_Non-Text Contrast: At 200% zoom, contrast ratio of keyboard focus on command bar with background is 1.6:1 which is less than required contrast ratio 3:1

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-docs#4512]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_Visual Studio Code#A11ySev2#A11yTCS#A11yWCAG2.1#Chrome#DesktopWeb#Win10#WCAG1.4.11#Non-Text Contrast#Zoom#Visual Studio Code Services#Benchmark-VisualStudioCode-Web-Apr2021

### Environment details:
Application Name: Visual Studio Code
URL:[Visual Studio Code](https://code.visualstudio.com/)
Windows Version: Version 20H2 (OS Build 19042.928)
Browser: Chrome Version 90.0.4430.85 (Official Build) (64-bit)

### Repro Steps:

1. Zoom the page to 200%
2. Hit [Visual Studio Code](https://code.visualstudio.com/) link
2. Navigate to command bar which is present in top navigation section
3. Verify that contrast ratio of keyboard focus on command bar with background is 1.6:1 which is less than required contrast ratio 3:1

### Actual Result:
Contrast ratio of keyboard focus on command bar with background is 1.6:1 which is less than required contrast ratio 3:1

### Expected Result:
Contrast ratio of keyboard focus on command bar with background should be greater than or equal to required contrast ratio 3:1

### User Impact:
Users with color blindness will not be able to identify where the focus is landing if contrast ratio is less than 3:1

### Attachment:
![A11y_Visual Studio Code_HomePage_Resize Text](https://user-images.githubusercontent.com/83570638/116961746-c47b1b80-acc1-11eb-8c42-d802244810c2.png)

**Comment by 12-shweta** [comment]
#A11yMAS;
